### ISSUE TITLE
Option: Disable statistics

### ISSUE THREAD

**Issue by Troelz** [MrZammler/iGame#48]
Please add an option to disable the saving of statistics / saving to gameslist on the start of every game.
Also a way to remove the Last played/Most played/Never played :)

Looking forward to more updates, and thanks for your effort!
- I sure hope you manage to finish an iGame V. 2.0 in the near future :)

**Comment by MrZammler** [comment]
Hi!

You can use the tooltype SAVESTATSONEXIT (or use the new settings window in 2.0 betas). This way, iGame will only commit stats on disk when you exit.

For the other, maybe an option to remove the stats alltogether would work. We'll see... :-)

thanks!

**Comment by Flak-Monkey** [comment]
Yes, please add option to disable all disk writes / run igame in READ ONLY mode.  Will help A570 users when running iGame from custom made CDROM's.

**Comment by midwan** [comment]
@MrZammler 
I think this could be easily handled by adding one more option in the Prefs, so it could look like this:
- Save Stats on Exit
- Save Stats immediately
- Disable Stats (or similar wording)

IMHO, there's no need to add another tooltype, since we started using Preferences with v2.

**Comment by MrZammler** [comment]
Yes, I was thinking it'll go somewhat like that. I've done some work on it already, will have it ready in a few days.
